docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,481 @@
+    "\n",
+    "This image has a **low** label quality score and is marked as an error. On closer inspection we notice the annotator missed a car in the back right corner that the model identified. Additionally, the bounding box for the foreground car the woman is in is cut short a bit on the right. \n",
+    "\n",
+    "Notice examples where the predictions and labels are more similar have higher quality scores than those that are missmatched, and are less likeley to be marked as issues and the number of boxes is agnostic to the score.\n",
+    "\n",
+    "Since our model is not perfect, it incorrectly identified the passenger dog as a person however it was still able to identify issues in the scene. Better trained models will lead to better label error detection but you don't need a near perfect model to identify label issues.\n",
+    "\n",
+    "\n",
+    "### Different kinds of label issues identified by ObjectLab\n",
+    "Now lets view the first 5 images in our vaidation dataset that are clearly marked as issues and see what other inconsistencies between the `given` and `predicted` label we can spot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94f82b0d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "example_issue_idx = issue_idx[:num_examples_to_show]\n",
+    "\n",
+    "issue_predictions = [predictions[idx] for idx in example_issue_idx]\n",
+    "issue_labels = [labels[idx] for idx in example_issue_idx]\n",
+    "issue_scores = scores[example_issue_idx]\n",
+    "\n",
+    "for idx, (prediction,label,score) in enumerate(zip(issue_predictions,issue_labels,issue_scores)):\n",
+    "    image_path = IMAGE_PATH + label['seg_map']\n",
+    "    print(image_path, '| idx', example_issue_idx[idx] , '| label quality score:', score, '| is issue: True')\n",
+    "    visualize(image_path, label=label, prediction=prediction, class_names=class_names, overlay=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03d5a521",
+   "metadata": {},
+   "source": [
+    "In each of these these examples there is a clear variety of issues. The corn-on-the-cob has a swapped label where a chair was identified as a person while the monitor picture is missing annotations for a cup and a person working on a chair in the background. The annotator of a kid's birthday party did not individually highlight each kid (in accordance to COCO guidelines only groups with 10 or more objects of the same type can be identified with a single box as a \"crowd\"). Some cups were forgotten at the dinner party and the empty chair was annotated incorrectly. Finally, the chairs are missing from the foreground of the abandoned school bus. \n",
+    "\n",
+    "All of these examples has low label quality scores to reflect their low annotation quality.\n",
+    "\n",
+    "### Other uses of visualize function\n",
```

Comment:
I'm finding this final section a little confusing, it took me a moment before understanding what you were trying to explain. 

A suggestion would be to move the text explaining what we are doing (eg. _"We can save the image by providing a `save_path` variable"_) directly above each code block instead of having the huge chuck of text then 3 back-to-back code blocks. 

I think we can also remove the _"For `idx=1`"_ note because I initially thought the index of the image was relevant. Instead we can just say something like _"For this image.."_ (and since we are doing on sentence of text per image it should be clear which image we are referencing)